### PR TITLE
Update service URLs and remove inaccessible services

### PR DESCRIPTION
**Pull request by cirxxle**
* Moved
  * SauceLabs free for Open Source (https://saucelabs.com/open-source): Moved to https://opensource.saucelabs.com/
  * cloudimage.com (https://cloudimage.com/)
    * https://performance.cloudimage.com/ shows this website is owned by Scaleflex (https://www.scaleflex.com)
    * In https://www.scaleflex.com/pricing, the new registration page is https://www.cloudimage.io/en/registration\
  * GeocodeAPI (https://geocodeapi.io)
    * Redirected to ipstack (https://ipstack.com/)
  * 3Dassets.one (https://www.3dassets.one/): Moved to https://3dassets.one/. SSL only for the root domain.

* Not free anymore
  * Restlet APIShark (https://restlet.com/products/apispark/)
    * Redirected to https://www.talend.com, didn't see APIShark on the page.
    * Talend Cloud (https://www.talend.com) only has a free trial for 14 days.
  * WayScript (https://www.wayscript.com): [Pricing](https://www.wayscript.com/pricing) page locked, no signup button on the home page.

* Unaccessible
  * vspace (https://vvv.space/): 403 Forbidden, [chrome extension](https://chrome.google.com/webstore/detail/vspace-%E2%80%94-free-bookmark-ma/poebjekpobpcmohnkbfkcfimilbeianl) hasn't been updated since October 25, 2021.
  * Similar Words API (https://word-simi.herokuapp.com/): 404 Not Found, Heroku returned "There's nothing here, yet.", possibly using the Free tier.
  * Hookbin (https://hookbin.com/): Timeout, couldn't connect to the server.
  * Krucible (https://usekrucible.com): Cloudflare returned an Origin DNS error, their [GitHub](https://github.com/Krucible) hasn't been updated since Dec 11, 2022.
  * Coin360 (https://api.coin360.com/): [Main page](https://coin360.com/) still works, API refused to connect (port 80/443 not open).
  * SETools.xyz (https://www.setools.xyz/)
    * CNAME record points to `test.setools.xyz.`
    * `test.setools.xyz.` DNS records are empty.
  * Unixtime (https://unixtime.co.za): Domain for sale
  * QR Code Robot (https://qr-code-robot.com): Domain for sale
  * Frontend Zero to One (https://www.fezto.xyz/): Could not resolve host
  * thundra.io/apm (https://www.thundra.io/apm): Could not resolve host
  * JWT Decoder (https://jwt.ssotools.com/): Could not resolve host
  * Issue Embed (https://issueembed.dev/): Could not resolve host
  * gitential.com (https://gitential.com): Could not resolve host
  * easyDB.io (https://easydb.io/): Could not resolve host
  * Sider (https://sider.review): Could not resolve host
  * Cmd (https://cmd.com/): Could not resolve host

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -582,7 +575,7 @@ This list is the result of Pull Requests, reviews, ideas and work done by 1100+
   * [loadmill.com](https://www.loadmill.com/) - Automatically create API and load tests by analyzing network traffic. Simulate up to 50 concurrent users for up to 60 minutes for free every month.
   * [preflight.com](https://preflight.com) - No-code automated web testing. Record tests on your browser that are resilient to UI changes and run them on Windows machines. Integrate with your CI/CD. The free plan includes 50 test runs per month with video, HTML sessions and more.
   * [percy.io](https://percy.io) - Add visual testing to any web app, static site, style guide, or component library.  Unlimited team members, Demo app and unlimited projects, 5,000 snapshots / month.
-  * [saucelabs.com](https://saucelabs.com/) — Cross browser testing, Selenium testing and mobile testing, [free for Open Source](https://saucelabs.com/open-source)
+  * [saucelabs.com](https://saucelabs.com/) — Cross browser testing, Selenium testing and mobile testing, [free for Open Source](https://opensource.saucelabs.com/)
```
Comment (ripienaar) [inline README.md:578]: This one I am not so sure about anymore, they dont say its free for open source, they seem to have some process for selecting which open source they support? So I am inclined to just remove it.

**Review by ripienaar — CHANGES_REQUESTED** [review]
Just one change otherwise this is great and thank you so much!

File: README.md (hunk 1 above)
Comment (cirxxle) [reply README.md:578]: They offer a Free Plan for everyone (2000 credits/week) and free licenses for open-source projects.

Comment (ripienaar) [reply README.md:578]: It says “free accounts for qualifying initiatives” not all. 

Everyone gets free credits weekly though so we should rather highlight their normal free tier and link to main website.

**Comment by ripienaar** [comment]
thanks a lot!

**Pull request merged**